search-app/src/cmr/search/services/parameters/converters/shapefile.clj

```diff
@@ -126,17 +129,17 @@
                     temp-dir)
           data-store (FileDataStoreFinder/getDataStore shp-file)
           feature-source (.getFeatureSource data-store)
-          collection (error-if (.getFeatures feature-source)
-                              #(< (.size %) 1)
-                              "Shapefile has no features"
-                              temp-dir)
-          _ (debug (format "Found [%d] features" (.size collection)))
-          iterator (.features collection)]
+          features (error-if (.getFeatures feature-source)
+                             #(< (.size %) 1)
+                             "Shapefile has no features"
+                             temp-dir)
+          _ (debug (format "Found [%d] features" (.size features)))
+          iterator (.features features)]
       (try
```

Comment:
Unrelated to your PR, but looks like a `try` within a `try` (lines 122 and 138)?